pkg/flare/archive.go

```diff
@@ -212,6 +219,27 @@ func zipDiagnose(zipFile *archivex.ZipFile, hostname string) error {
 	return nil
 }
 
+func zipHealth(zipFile *archivex.ZipFile, hostname string) error {
+	s := health.GetStatus()
+	if len(s.Healthy) > 0 {
+		sort.Strings(s.Healthy)
+	}
+	if len(s.Unhealthy) > 0 {
+		sort.Strings(s.Unhealthy)
+	}
+
+	yamlValue, err := yaml.Marshal(s)
+	if err != nil {
+		return err
+	}
+	err = zipFile.Add(filepath.Join(hostname, "health.yaml"), yamlValue)
```

Comment:
For consistency, why don't use a common for func for the output on the CLI & on the file (taking a writer in param)?

Reply:
The flare will be read in a text editor, where I'd rather have it as a vertical list, while I think the horizontal list makes more sense in the console cmd. WDYT?